tests/renderer/remote-loader-spec.ts

```diff
@@ -165,6 +165,29 @@ describe('RemoteLoader', () => {
       expect(store.setVersion).toBeCalledWith('17.0.0');
     });
 
+    it('handles gists with files over 1mb', async () => {
+      const gistId = 'toobig';
+
+      mockGistFiles['index.js'] = {
+        truncated: true,
+        content: JSON.stringify('hello im huge', null, 2),
+        raw_url: 'https://gist.githubusercontent.com/IMTOOBIG',
+      };
+
+      jest.spyOn(global, 'fetch').mockImplementationOnce(() =>
+        Promise.resolve({
+          text: () => Promise.resolve('hello im huge'),
+        } as Response),
+      );
+
+      mocked(getOctokit).mockResolvedValue({
+        gists: mockGetGists,
+      } as unknown as Octokit);
+
+      const result = await instance.fetchGistAndLoad(gistId);
+      expect(result).toBe(true);
+    });
```

Comment:
Some tweaks are needed for the test, unfortunately GitHub doesn't show a pretty diff here.

The test before would also pass if you reverted the changes in `src/renderer/remote-loader.ts`. The suggested tweaks:
* Ensure the truncated content differs from the response to `raw_url` so we know it got the correct content
* Makes sure `instance.confirmAddFile` is mocked to true, since the filename is not one of the standard files
* Adds the expected contents to `editorValues` and adds an `expect(app.replaceFiddle)` at the end

```suggestion
    it('handles gists with files over 1mb', async () => {
      const gistId = 'toobig';
      const filename = 'index.js';
      const content = 'hello im huge';

      editorValues[filename] = content;
      mockGistFiles[filename] = {
        truncated: true,
        content: 'truncated',
        raw_url: 'https://gist.githubusercontent.com/IMTOOBIG',
      };

      jest.spyOn(global, 'fetch').mockResolvedValueOnce({
        text: () => Promise.resolve(content),
      } as Response);

      mocked(getOctokit).mockResolvedValue({
        gists: mockGetGists,
      } as unknown as Octokit);
      instance.confirmAddFile = jest.fn().mockResolvedValue(true);

      const result = await instance.fetchGistAndLoad(gistId);
      expect(result).toBe(true);
      expect(app.replaceFiddle).toBeCalledWith(editorValues, { gistId });
    });
```

Reply:
> Some tweaks are needed for the test, unfortunately GitHub doesn't show a pretty diff here.
> 
> The test before would also pass if you reverted the changes in `src/renderer/remote-loader.ts`. The suggested tweaks:
> 
> * Ensure the truncated content differs from the response to `raw_url` so we know it got the correct content
> * Makes sure `instance.confirmAddFile` is mocked to true, since the filename is not one of the standard files
> * Adds the expected contents to `editorValues` and adds an `expect(app.replaceFiddle)` at the end

Applied changes, thank you!